libs/wingsdk/src/core/app.ts

```diff
@@ -49,6 +50,12 @@ export interface AppProps {
    * @default - use the default polycon factory included in the Wing SDK
    */
   readonly customFactory?: IPolyconFactory;
+
+  /**
+   * Plugins to apply on app.
```

Comment:
Describe what are the strings here? (They are paths to JavaScript files). Are they relative/absolute. If relative, relative to what?

(The safest option is to require that paths are always absolute at this layer and have the cli resolve any relative paths against cwd)